### PR TITLE
New selection architecture

### LINKED ISSUE #3: Sprint Selection
As a user,
I want to select one sprint,
So that the stories contained is displayed

### PR DESCRIPTION
**Pull request by lucas-barbero**

### CODE REVIEW

File: RunningOrder/Sprint/SprintList.swift [hunk 1]
```diff
@@ -22,18 +22,22 @@ struct SprintList: View {
...
-                }
-
-                Button(action: { self.showNewSprintModal.toggle() }) {
-                    Text("New Sprint")
+                    NavigationLink(
+                        destination: StoryList(stories: sprint.stories)
+                            .listStyle(PlainListStyle()),
+                        label: {
+                            Text(sprint.name)
+                        }).tag(sprint)
```
Comment (bilalb) [inline RunningOrder/Sprint/SprintList.swift]: what's the goal of the tag?

File: RunningOrder/Story/StoryDetail.swift [hunk 2]
```diff
@@ -9,13 +9,19 @@
 import SwiftUI
 
 struct StoryDetail: View {
+    let story: Story
+
     var body: some View {
-        Text("Hello World!").padding()
+        VStack(alignment: .center) {
```
Comment (bilalb) [inline RunningOrder/Story/StoryDetail.swift]: As `.center` is the default alignment value, you can ommit that part :)

**Review by bilalb — APPROVED** [review]

**Review by zarghol — APPROVED** [review]

File: RunningOrder/Sprint/SprintList.swift (hunk 1 above)
Comment (lucas-barbero) [reply RunningOrder/Sprint/SprintList.swift]: Useless here indeed because `selectedSprint`is not used anymore

Comment (lucas-barbero) [reply RunningOrder/Sprint/SprintList.swift]: Useless here indeed, because `selectedSprint` is not used anymore

**Pull request merged**